cot/src/admin.rs

```diff
@@ -564,7 +637,7 @@ impl App for AdminApp {
             ),
             crate::router::Route::with_handler_and_name("/login/", login, "login"),
             crate::router::Route::with_handler_and_name(
-                "/{model_name}/",
+                "/{model_name}",
```

Comment:
Why this change? I think we should have consistent URLs across the admin app (we have /login**/**, /create**/**, etc.).

Reply:
reverted back, at first I was not able to get it to work not sure why